Backpack/src/main/res/values-night/internal.semantic.color.xml

```diff
@@ -20,9 +20,11 @@
   <color name="__switchThumbDisabled">@color/bpkBlackTint05</color>
   <color name="__starRatingStarColor">@color/bpkBlackTint05</color>
   <color name="__horizontalNavTextDefault">@color/bpkTextTertiaryDark</color>
-  <color name="__textFieldText">@color/bpkTextPrimaryDark</color>
-  <color name="__textFieldHint">@color/bpkTextTertiaryDark</color>
-  <color name="__textFieldIcon">@color/bpkTextSecondary</color>
+  <color name="__textFieldTextDisabled">@color/bpkSkyGrayTint02</color>
+  <color name="__textFieldHint">@color/bpkSkyGrayTint04</color>
```

Comment:
Can we avoid using skyGray in dark mode?
It looks better if we use the black tints instead.

Reply:
It's in the design document. We can raise a request, I also think it looks better without skyGrays.

Reply:
One of Will's rules for dark mode was that skyGray shouldn't be used, but I don't think we ever wrote that down anywhere.
Whenever I see it in product I die a little bit inside 😆

Reply:
In the screenshots it looks like this isn't gray any more, but the code still says `bpkSkyGrayTint04` 🤔